### PR TITLE
[docs] Add steps for installing chart with image built from local branch

### PR DESCRIPTION
**Pull request by jsanda**

### CODE REVIEW

File: documentation/development.md [hunk 1]
```diff
@@ -122,6 +122,44 @@ LOG_LEVEL=Debug
...
+Build the image from the current branch.
+
+```
+$ export DOCKER_REPO_BASE=jsanda
+$ make docker-build
+```
+Push the image to docker hub (or to whichever repo you want to use)
+
+```
+$ docker push jsanda/cassandra-k8s-operator:0.3.1-local-dev-helm
```
Comment (Orange-cscetbon) [inline documentation/development.md]: `docker push $DOCKER_REPO_BASE/cassandra-k8s-operator:0.3.1-local-dev-helm`

File: documentation/development.md [hunk 2]
```diff
@@ -122,6 +122,44 @@ LOG_LEVEL=Debug
 OPERATOR_NAME=ide
 ```
 
+#### Deploy using the Helm Charts
+This section provides an instructions for running the operator Helm charts with an image that is built from the local branch.
+
+Build the image from the current branch.
+
+```
+$ export DOCKER_REPO_BASE=jsanda
```
Comment (cscetbon) [inline documentation/development.md]: I propose to use `orangeopensource` here. Anyone can change it to the account they want but that's the one we use to store our images for instance

**Comment by Orange-cscetbon** [comment]
@jsanda, please resolve the conflicts

**Pull request merged**